aiosmtpd/docs/controller.rst

```diff
@@ -169,7 +169,8 @@ Controller API
 
 .. class:: Controller(handler, loop=None, hostname=None, port=8025, *, ready_timeout=1.0, enable_SMTPUTF8=None, ssl_context=None, server_kwargs=None)
 
-   **Parameters**
+   |
+   | :part:`Parameters`
```

Comment:
We use "line blocks" to give vertical spacing. I can't figure out how to give a gap there using CSS; nothing I tried works.